### PR TITLE
Sof 823/evs shortcuts incorrectly configured

### PR DESCRIPTION
**Pull request by KvelaGorrrrnio**
Default shortcuts for EVS (queue and cut_to_box) have been updated.

It is dependent on https://github.com/nrkno/sorensen/pull/2.

### CODE REVIEW

File: src/tv2-common/hotkeys/local.ts [hunk 1]
```diff
@@ -11,15 +11,29 @@ function localSourceFullAudioHotkeyId(showStyleId: string, sourceLayer: string,
...
 function localSourceVoAudioHotkeyId(showStyleId: string, sourceLayer: string, hotkeyType: string, index: number) {
 	return `${showStyleId}_${sourceLayer}_local_vo_audio_${hotkeyType}_${index}`
 }
 
 function localSourceVoAudioHotkeyName(source: string) {
-	return `EVS ${source}`
+	return `EVS ${source} VO`
+}
+
+function localSourceVoAudioPick(sourceIndex: number, hotkeyType: string): number {
```
Comment (skon-tv2) [inline src/tv2-common/hotkeys/local.ts:32]: I don't really understand what this function does. What is the condition and where does the magic numbers come from and why!?

Comment (KvelaGorrrrnio) [reply src/tv2-common/hotkeys/local.ts:32]: I assume that you are referring to `sourceIndex` in `localSourceVoAudioPick`.
If we use the following snippet as our example, then the index of the shortcut is used as the sourceIndex to
https://github.com/tv2/sofie-blueprints-inews/blob/bd925525782027969c3812b427aee8d6adddad68/src/tv2-common/hotkeys/hotkey-defaults.ts#L107

The problem before was that multiple items targeted the same sequence of actions with the same indices. I have introduced the ability to transform the target for the pick-filter, based on the position in the list and the type of trigger it is ('queue', 'cut_to_box_XX', etc.).

**Review by LindvedKrvang — APPROVED** [review]

File: src/tv2-common/hotkeys/local.ts [hunk 2]
```diff
@@ -11,15 +11,29 @@ function localSourceFullAudioHotkeyId(showStyleId: string, sourceLayer: string,
 }
 
 function localSourceFullAudioHotkeyName(source: string) {
-	return `EVS ${source}`
+	return `EVS ${source} 100%`
+}
+
+function localSourceFullAudioPick(sourceIndex: number, hotkeyType: string): number {
+	if (hotkeyType === 'queue' || /^cut_to_box_\d+$/.test(hotkeyType)) {
```
[the remark this answers is not in the document]
Comment (KvelaGorrrrnio) [reply src/tv2-common/hotkeys/local.ts]: @skon-tv2 Duplicated code has been eliminated.

File: src/tv2-common/hotkeys/local.ts [hunk 3]
```diff
@@ -11,15 +11,33 @@ function localSourceFullAudioHotkeyId(showStyleId: string, sourceLayer: string,
...
-	return `EVS ${source}`
+	return `EVS ${source} VO`
+}
+
+function localSourceVoAudioPick(sourceIndex: number, hotkeyType: string): number {
+	if (shouldUseCustomIndexPickerForLocalSourceAudio(hotkeyType)) {
+		return 1 + 2 * sourceIndex
+	}
+	return sourceIndex
+}
```
Comment (ianshade) [inline src/tv2-common/hotkeys/local.ts:37]: I'm not a fan of this. Can't we use tags, and then always just pick the sourceIndex-th adlib?

Comment (KvelaGorrrrnio) [reply src/tv2-common/hotkeys/local.ts:37]: In the current setup, the same tag is used for multiple lists with the same index, which is why I did it this way.
I can have a look at how to do what you suggest.

**Review by ianshade — APPROVED** [review]
Well done

**Pull request merged**